core/chains/evm/client/rpc_client.go

```diff
@@ -134,14 +107,11 @@ type rpcClient struct {
 
 	// Need to track subscriptions because closing the RPC does not (always?)
 	// close the underlying subscription
-	subs []ethereum.Subscription
-
-	// Need to track the aliveLoop subscription, so we do not cancel it when checking lease on the MultiNode
-	aliveLoopSub ethereum.Subscription
+	subs map[ethereum.Subscription]struct{}
 
 	// chStopInFlight can be closed to immediately cancel all in-flight requests on
-	// this rpcClient. Closing and replacing should be serialized through
-	// stateMu since it can happen on state transitions as well as rpcClient Close.
+	// this RpcClient. Closing and replacing should be serialized through
```

Comment:
nit:

I notice in the code it's always `RPCClient`, but in the comments it's `RpcClient` in most places, and still `rpcClient` in a few places.

Reply:
Updated comments to all be "RPCClient"